net.sourceforge.atanks.appdata.xml

```diff
@@ -6,6 +6,7 @@ SentUpstream: 2014-09-17
 -->
 <application>
   <id type="desktop">atanks.desktop</id>
```

Comment:
it'd be really great to update this to a modern app-id as well, and include this name as a launchable for backwards compatiblity.